srtcore/api.cpp

```diff
@@ -124,7 +124,10 @@ SRT_SOCKSTATUS CUDTSocket::getStatus()
     return m_Status;
 }
 
-void CUDTSocket::makeClosed()
+/// This makes the socket no longer capable of performing any transmission
+/// operation, but continues to be responsive in the connection in order
+/// to finish sending the data that were scheduled for sending so far.
+void CUDTSocket::makeShutdown()
```

Comment:
I would suggest placing this description at the function declaration (api.h) instead of the function definition.

Reply:
Also, I find your earlier description more clear.
@stevomatthews Could you please advise on this?

> Shutdown makes it no longer possible to perform send/recv operation on the socket, but REMAINING DATA from the previous requests continue to be performed until the queues are empty or until the linger expires.

Reply:
You copied the text together with my mistake :D